test/cpp/interop/xds_interop_client.cc

```diff
@@ -470,6 +494,14 @@ void BuildRpcConfigsFromFlags(RpcConfigurationsQueue* rpc_configs_queue) {
     if (metadata_iter != metadata_map.end()) {
       config.metadata = metadata_iter->second;
     }
+    if (absl::GetFlag(FLAGS_request_payload_size) > 0) {
+      config.request_payload_size = absl::GetFlag(FLAGS_request_payload_size);
+      std::string payload(config.request_payload_size, '0');
+      config.request_payload = payload;
+    }
+    if (absl::GetFlag(FLAGS_response_payload_size) > 0) {
```

Comment:
Instead, let's add validation that the flag value is not negative, and exit via `GPR_ASSERT(0);` it's not.

Reply:
Done